packages/@lightningjs/ui-components/src/utils/index.js

```diff
@@ -854,6 +854,16 @@ export function convertTextAlignToFlexJustify(align) {
   }
 }
 
+/**
+ * Write params in here soon
+ * function is checking for single num or array
+ */
+export function getMaxRoundRadius(radius, width, height, offset){
+  const maxRadius = r => Math.min(radius, (Math.min(width, height) / 2)) + offset;
```

Comment:
Change to const maxRadius = r => Math.min(r, (Math.min(width, height) / 2)) + offset;

Reply:
It also might be useful to cast to a number 

export function getMaxRoundRadius(radius, width, height, offset) {
  const maxRadius = r => Math.min(r, Math.min(width, height) / 2) + offset;
  return Array.isArray(radius)
    ? radius.map(r => Number(maxRadius(r)) || 0)
    : Number(maxRadius(radius)) || 0;
}